check-peer-deps.js

```diff
@@ -167,8 +169,33 @@ const checkAllPeerDeps = async () => {
   return Promise.all(promises);
 };
 
+const setOpts = () => {
+  const DEBUG_DEFAULT = false;
+  const NO_INCLUDE_DEV_DEFAULT = false;
+  const MAX_RETRIES_DEFAULT = 2;
+
+  const opts = commandLineArgs([
+    { name: 'debug', alias: 'd', type: Boolean, defaultValue: DEBUG_DEFAULT },
+    { name: 'no-include-dev', type: Boolean, defaultValue: NO_INCLUDE_DEV_DEFAULT },
+    { name: 'max-retries', type: Number, defaultValue: MAX_RETRIES_DEFAULT },
+    { name: 'help', alias: 'h', type: Boolean }
+  ]);
+
+  DEBUG = opts['debug'] ? opts['debug'] : DEBUG_DEFAULT;
```

Comment:
Specifying the `defaultValue` above makes this section the same as setting the values equal to `opts['name']`.